packages/tupaia-web/src/features/Map/MapOverlaySelector/DesktopMapOverlaySelector.tsx

```diff
@@ -146,47 +164,59 @@ export const DesktopMapOverlaySelector = ({
 }: DesktopMapOverlaySelectorProps) => {
   const { projectCode, entityCode } = useParams();
   const { hasMapOverlays } = useMapOverlays(projectCode, entityCode);
-  const { period } = useActiveMapOverlayReport();
-
+  const { data: mapOverlayData } = useMapOverlayReport();
```

Comment:
Just checking if it's intentional that this was changed from `const { period } = useActiveMapOverlayReport();` to this?